### DISCUSSION TITLE
use of image MetaTensor.meta

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1313] [Q&A]
Hi there!

I'm confused about the use of image MetaTensor.meta.

After running LoadImage or LoadImaged, the metadata will be loaded into header dict, correct? Then what's the purpose of the returned image MetaTensor.meta?

Both array and dictionary transforms can return image MetaTensor. Does MetaTensor.meta record all the transform matrices? If it is, can we use it to invert the transform chain to recover the original image? Any tutorial showing how to do it?

**Comment by KumoLiu** [comment]
Hi @hw-ju, thanks for your interest here!
The meta information will be saved in `data.meta`.
And for more information about how to use `data.meta` or how to invert the transforms, you could refer to these tutorials:
https://github.com/Project-MONAI/tutorials/blob/main/modules/inverse_transforms_and_test_time_augmentations.ipynb
https://github.com/Project-MONAI/tutorials/blob/main/modules/transforms_metatensor.ipynb

Hope it can help you, thanks!

**Reply by hw-ju** [reply]
Thanks a lot @KumoLiu! I'll take a look!
